spec/features/users_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+feature "can sign up for an account" do
+  scenario "with valid attributes" do
+    visit new_user_path
+
+    fill_in "user_name", with: "Mary"
+    fill_in "user_email", with: "Mary@example.com"
+    fill_in "user_password", with: "secret"
+    click_on I18n.t("users.new.submit")
+
+    expect(page).to have_text(I18n.t("users.show.welcome", name: "Mary"))
```

Comment:
If you wish, you can check that the page displays the flash[:notice]

Reply:
Thanks Elle, I have added this to the update spec as well, I think it makes it more consistent with the sad path specs!

Reply:
A suggestion:
```
feature "user creates new account" do
  scenario "successfully" do
  end
end
```

And the file name matches the describe text

Reply:
Done with sha: db61cc7fd67c2313affa2e9df0f57363516bb1d0